app/views/angular/boot.js.erb

```diff
@@ -0,0 +1,21 @@
+var includeJs = function(url, callback) {
+  var elem   = document.createElement('script')
+  elem.src   = url
+  elem.onload = callback
+  document.getElementById('boot').appendChild(elem)
+}
+
+var includeCss = function(url) {
+  var elem   = document.createElement('link')
+  elem.href  = url
+  elem.media = 'screen'
+  elem.rel   = 'stylesheet'
+  document.getElementById('boot').appendChild(elem)
+}
+
```

Comment:
I like having these all bunched together like this into a single boot file; makes it really obvious what scripts show up and why